packages/grafana-runtime/src/components/PluginBridge.test.tsx

```diff
@@ -0,0 +1,49 @@
+import { screen, render } from '@testing-library/react';
+import React from 'react';
+
+// @ts-ignore
+import { backendSrv } from 'app/core/services/backend_srv';
```

Comment:
We're using `@ts-ignore` here because it doesn't know how to resolve this path unless the test suite is ran from the project root.

I don't particularly like this part because we're importing code from outside this package, but since it's only for the test suite and I didn't want to re-implement `BackendSrv` I've decided to keep it this way.